rtxdi-sdk/include/rtxdi/ResamplingFunctions.hlsli

```diff
@@ -1563,6 +1571,9 @@ RTXDI_Reservoir RTXDI_SpatialResampling(
         RAB_LightSample candidateLightSample = RAB_EmptyLightSample();
         if (RTXDI_IsValidReservoir(neighborSample))
         {   
+            if (sparams.discountNaiveSamples && neighborSample.M <= RTXDI_NAIVE_SAMPLING_M_THRESHOLD)
+                continue;
```

Comment:
This logic needs to be implemented in all the spatiotemporal function variations.

Reply:
I have added this